packages/terra-core-docs/src/terra-dev-site/test/pills/pill/BasicRemovablePill.test.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { useState } from 'react';
+import classNames from 'classnames/bind';
+import { Pill } from 'terra-pills';
+import styles from '../pill-list/PillListTestCommon.module.scss';
+
+const cx = classNames.bind(styles);
+
+const BasicRemovablePill = () => {
+  const [isRemoved, setIsRemoved] = useState(true);
```

Comment:
nit : can we change this to `const [isRemoved, setIsRemoved] = useState(false);` and modify to return to have `!isRemoved` instead of `isRemoved`.